x-pack/plugins/apm/public/components/app/service_map/__stories__/cytoscape_example_data.stories.tsx

```diff
@@ -105,7 +105,7 @@ export const GenerateMap: Story<{}> = () => {
       </Cytoscape>
 
       {json && (
-        <CodeEditor
+        <CodeEditor // TODO Unable to find context that provides theme. Need CODEOWNER Input
```

Comment:
I think you need to add the necessary context when invoking `MockApmPluginStorybook`:

https://github.com/elastic/kibana/blob/ca425e899344a6381e05113c93f861dd44201d86/x-pack/plugins/apm/public/components/app/service_map/__stories__/cytoscape_example_data.stories.tsx#L49

Something like this:
```
<MockApmPluginStorybook apmContext={{ foo: 'bar' }}>
```

This is passed on to the Kibana Context Provider here:
https://github.com/elastic/kibana/blob/8861c27f44e70e8853e508887ee599e8f4a4341e/x-pack/plugins/apm/public/context/apm_plugin/mock_apm_plugin_storybook.tsx#L136-L140

Reply:
Sorry, I'm struggling to figure this out; can you link me to where the `GenerateMap` and `MapFromJson` story functions are being used?

Reply:
It's used for storybook: https://ci-artifacts.kibana.dev/storybooks/main/eac8f4ce25737e7f2dab5d9d131e2a414c4fa47b/apm/index.html?path=/story/app-servicemap-example-data--map-from-json

You can run it locally with `yarn storybook apm`

Reply:
@sqren based on the Slack discussion it seems these CodeEditors (in Storybook) might have other issues going on. Would it be alright to move forward with this PR and address them at a later date?

Reply:
Yes, fine to ignore the storybook failures for now. Sorry for being slow to follow up on this.

Reply:
No worries! Thanks for the update.